### ISSUE TITLE
Library stopped working

### ISSUE THREAD

**Issue by alicja-lachman** [cortinico/myonnaise#10]
Hello, 
Is it possible that the library stopped working? Neither the discovery nor the connection are working right now. I use Myonnaise in my school project and it was working perfectly fine, now after some break it doesn't seem to work - as well as your sample app Myo EMG Visualiser. At first I thought it was a problem with my phone or with my Myo, but Bluetooth discovery works fine and another app (2048 Myo Edition) also connects with the band and works.

**Comment by cortinico** [comment]
Hey @alicja-lachman 
Can I ask you to provide further details? Do you have a project that I can test?
Are you using the version `1.0.0` of the library?
I haven't released any other version so it looks weird to me that the library stopped working.

Generally if the search is not working, could be that you revoked the Location permission. Can you please check that you have that location granted?

**Comment by alicja-lachman** [comment]
Hey @cortinico! Thank you for the quick answer! Yes, I was using the version 1.0.0 of the library, in the code where I have used it before with no problems. Today it stopped working in my app and also your sample app from Play Store. Funny thing - I built and installed TouchSense sample app (probably you know it, as there is your name on the main screen) where it asked me to accept location permission (I was doing so also in my app, but this time instead of only accepting it on the pop-up, I had to do it in the settings) and boom - now it's working. TouchSense, Myo EMG Visualizer and also my app. This was really strange, maybe it was a glitch on my device. Anyway, so sorry to bother you, but big thanks for quick reply!
